interfaces/automated-testing/scd/design/astm-strategic-coordination/nominal-planning-priority/sequence.plantuml

```diff
@@ -20,9 +20,22 @@ TD <- USS1: 200 Ok - returns Operational Intent 1
 
 ...
 
-== TD injects via Second USS a conflicting Operational Intent 2 in space and/or time with a higher priority ==
+== TD injects via Second USS a conflicting flight 2 in space and/or time with a higher priority ==
 
-TD -> USS2: PUT /inject/flight
+TD -> USS2: PUT /inject/flight\n{"state": "Accepted"}
+TD <- USS2: 200 Ok - Operational Intent 2 created\n{"result": "Planned"}
+
+TD -> DSS: POST /dss/v1/operational_intent_references/query\nwith area_of_interest encompassing Operational Intent 2
+TD <- DSS: 200 Ok - returns Operational Intent Reference 2 with entityid2 managed by Second USS
+
+TD -> USS2: GET /uss/v1/operational_intents/{entityid2}
+TD <- USS2: 200 Ok - returns Operational Intent 2
+
+...
+
+== TD transitions flight in Second USS to Activated ==
+
+TD -> USS2: PUT /inject/flight\n{"state": "Activated"}
 TD <- USS2: 200 Ok\n{"result": "Planned"}
```

Comment:
Should we introduce a new result value to reflect actived flights?

Reply:
Yes, that seems to make sense; will do.